### PR TITLE
docs: 0.13 update for docker compose migration guide

### PR DESCRIPTION
**Pull request by vvagaytsev**
**What this PR does / why we need it**:
Related example update: https://github.com/garden-io/garden-docker-compose/pull/2

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/guides/migrating-from-docker-compose.md [hunk 1]
```diff
@@ -36,129 +43,144 @@ providers:
...
+      path: /api
+      port: http
+  ingresses:
+    - path: /
+      port: http
+      hostname: backend.${var.base-hostname}
 ```
 
-This one is a `Module` instead of a `Project`. Under `sync`, we set up syncing from the root folder to our built `app` folder so we can see code changes live when in developer mode. Under `ports`, we specify the same port as in our Docker Compose file (3000). We set up a health check for the the `/api` route, as that is what this module serves, and an ingress on a subdomain. In our case, this will let us access our backend service on `compose2garden.local.demo.garden`. Finally, we specify the dependency on the `mongo` module, which we will define in a bit.
+It defines 2 actions: a trivial `Build` action, and a `Deploy` action.
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:89]:
```suggestion
A `Build` action and a `Deploy` action are defined.
```

File: docs/guides/migrating-from-docker-compose.md [hunk 2]
```diff
@@ -36,129 +43,144 @@ providers:
...
+Let's consider `Deploy` action in a bit more details.
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:90]:
```suggestion
Make note of the `Deploy` action and it's configuration.
```

File: docs/guides/migrating-from-docker-compose.md [hunk 3]
```diff
@@ -36,129 +43,144 @@ providers:
...
+
+Under `sync` we set up syncing from the root folder to our built `app` folder, so we can see code changes live when in
+developer mode.
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:93]:
```suggestion
Under `sync` we set up syncing from the action root to the `app` folder on the container, so we can synchronize code changes live when in
`sync` mode.
```

File: docs/guides/migrating-from-docker-compose.md [hunk 4]
```diff
@@ -36,129 +43,144 @@ providers:
...
+
+Under `ports` we specify the same port as in our Docker Compose file (`3000`).
+
+We set up a health check for the `/api` route, as that is what this module serves, and an ingress on a subdomain. In our
+case, this will let us access our `backend` application on `compose2garden.local.demo.garden`.
+
+Finally, we specify the dependency on the `mongo` module, which we will define in a bit.
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:100]: Modules have dropped out of style

File: docs/guides/migrating-from-docker-compose.md [hunk 5]
```diff
@@ -36,129 +43,144 @@ providers:
...
 
 ### The `frontend/frontend.garden.yml` file
 
-For the frontend, we create another Garden module with the following contents:
+For the `frontend` we create another Garden configuration file:
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:104]:
```suggestion
For the `frontend` application we create separate Garden configuration file:
```

File: docs/guides/migrating-from-docker-compose.md (hunk 3 above)
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:93]:
```suggestion
Under `sync` we set up syncing from the action root to the `app` folder on the container, so we can synchronize code changes live when in
`sync` mode.
```

File: docs/guides/migrating-from-docker-compose.md [hunk 6]
```diff
@@ -36,129 +43,144 @@ providers:
...
 ```
 
-This is similar to the backend module, but we specify the backend as a dependency, which makes the database (`mongo`) an indirect dependency.
+This is similar to the `backend` application, but we specify the `backend` deployment as a dependency, which makes
+the database (`mongo`) an indirect dependency.
 
 ### The `mongo/mongo.garden.yml` file
 
-Here we've created a `mongo` folder, as it did not exist as an explicit module in our original Docker Compose project. The folder contains only the Garden configuration file, which contains:
+Here we've created a `mongo` folder, as it did not exist as an explicit module in our original Docker Compose project.
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:149]: Module

File: docs/guides/migrating-from-docker-compose.md [hunk 7]
```diff
@@ -36,129 +43,144 @@ providers:
...
-      - name: data
-        containerPath: /data/db
-    ports:
-      - name: db
-        containerPort: 27017
-```
 
-This specifies the same volume and port that we previously specified in Docker Compose.
-
-## Running a development version of the project on Garden
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:133]: I think you should add a note about `garden dev` and `sync` mode somewhere as a replacement for this block.

File: docs/guides/migrating-from-docker-compose.md [hunk 8]
```diff
@@ -36,129 +43,144 @@ providers:
...
 
+Use `frontend` application's ingress URL from the console output to open the application.
+
 ## Larger migrations
 
-This was a simple example, but it should give you what you need to migrate larger projects too. If you have feedback on how we could make migrating from Docker Compose easier, please send us feedback via [GitHub issues](https://github.com/garden-io/garden/issues), or reach out on [our Discord community](https://discord.gg/FrmhuUjFs6).
+This was a simple example, but it should give you what you need to migrate larger projects too. If you have feedback on
+how we could make migrating from Docker Compose easier, please send us feedback
+via [GitHub issues](https://github.com/garden-io/garden/issues), or reach out
+on [our Discord community](https://discord.gg/FrmhuUjFs6).
```
Comment (Orzelius) [inline docs/guides/migrating-from-docker-compose.md:185]:
```suggestion
This is a basic example but it should give you what you need to migrate larger projects too. If you have feedback on
how we could make migrating from Docker Compose easier, please send it our way
via [GitHub issues](https://github.com/garden-io/garden/issues) or reach out
on [our Discord community](https://discord.gg/FrmhuUjFs6).
```

I don't really like the "should give you what you need to" part but couldn't think of anything better.

File: docs/guides/migrating-from-docker-compose.md (hunk 3 above)
Comment (vvagaytsev) [reply docs/guides/migrating-from-docker-compose.md]: This has been committed.

File: docs/guides/migrating-from-docker-compose.md (hunk 7 above)
Comment (vvagaytsev) [reply docs/guides/migrating-from-docker-compose.md:133]: Yes, we can put a link to the code sync guide here, and not describe too many details.

Comment (vvagaytsev) [reply docs/guides/migrating-from-docker-compose.md:133]: Restored it in https://github.com/garden-io/garden/pull/4235/commits/f9ff0fdde7af965b451bc0f854c4873e33b8bc1d

File: docs/guides/migrating-from-docker-compose.md (hunk 6 above)
Comment (vvagaytsev) [reply docs/guides/migrating-from-docker-compose.md]: Changed this in https://github.com/garden-io/garden/pull/4235/commits/f1e1c0cbc989c1e828031c7fe65db59ae0c947ba

**Review by Orzelius — APPROVED** [review]

**Pull request merged**